### PR TITLE
Typography and lists

### PR DESCRIPTION
**Pull request by parvez-rashid**
View the screenshots below, will include preview link soon.

image
image
image
image
image

### CODE REVIEW

**Comment by Naman-Saxena1** [comment]
Code looks fine.
You can add screenshots in PR description as preview link is not working and it is also a good practice to add screenshots in PR.

**Comment by parvez-rashid** [comment]
> Code looks fine. You can add screenshots in PR description as preview link is not working and it is also a good practice to add screenshots in PR.

Sure, will do.

**Review by Naman-Saxena1 — APPROVED** [review]
Code and preview screenshots looks great 👍

File: list/list.html [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="/list/list.css" />
+    <title>Document</title>
+  </head>
+  <body>
+    <div class="example-container">
+      <h3 class="h3">Unordered lists</h3>
+      <div class="list-container">
+        <ul class="text-size-md">
+          <li>List item-1</li>
```
Comment (Ankur9669) [inline list/list.html:15]: The ul tags have some user agent style sheet styling so try to reset it

**Review by Ankur9669 — CHANGES_REQUESTED** [review]
image
These headings have a lot of margin are those intentional.

File: list/list.html (hunk 1 above)
Comment (parvez-rashid) [reply list/list.html:15]: This is a preview for just the components as of now, will work on styling these after I merge with the home page in dev. Will take care of the margin and user-agent stylesheet.

Comment (Ankur9669) [reply list/list.html:15]: Sure

**Review by Ankur9669 — APPROVED** [review]

**Pull request merged**